hack/gen-swagger-doc/run-gen-swagger-docs.sh

```diff
@@ -18,9 +18,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
-V1_PATH="$PWD/${KUBE_ROOT}/docs/api-reference/v1/"
-V1BETA1_PATH="$PWD/${KUBE_ROOT}/docs/api-reference/extensions/v1beta1"
+if [ "$#" -lt 1 ]; then
+    echo "Usage: run-gen-swagger-docs.sh <absolute output path>"
+    exit
+fi
+OUTPUT=${1:-}
```

Comment:
Why not setting the default value for OUTPUT to "$PWD/${KUBE_ROOT}/docs/api-reference/"? That will be convenient.